orc/src/main/java/org/apache/iceberg/orc/ORCSchemaUtil.java

```diff
@@ -214,10 +212,13 @@ public static Schema convert(TypeDescription orcSchema) {
         "Error in ORC file, children fields and names do not match.");
 
     List<Types.NestedField> icebergFields = Lists.newArrayListWithExpectedSize(children.size());
-    AtomicInteger lastColumnId = new AtomicInteger(getMaxIcebergId(orcSchema));
```

Comment:
I think the javadoc on this method needs revision?

Reply:
Done.